src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -101,12 +101,17 @@ const MSG = defineMessages({
     payment.`,
   },
   userPickerPlaceholder: {
-    id: 'SingleUserPicker.userPickerPlaceholder',
+    id: `dashboard.CreatePaymentDialog.CreatePaymentDialogForm.userPickerPlaceholder`,
     defaultMessage: 'Search for a user or paste wallet address',
   },
+  warningText: {
+    id: `dashboard.CreatePaymentDialog.CreatePaymentDialogForm.warningText`,
+    defaultMessage: `<span>Warning.</span> You are about to make a payment to an address not on the whitelist. Are you sure the address is correct?`,
```

Comment:
Not a big of putting the tags in the message descriptors text, but I guess it's fine since you're overwriting the chunk in the values